prep-javascript-dom-manipulation/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Manipulation</title>
+  <style>
+    #container {
+      border: 2px solid green;
+    }
+  </style>
+</head>
+
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="second-paragraph">
+
+    </p>
+    <p class="third-paragraph">
+
+    </p>
+  </div>
+  <script>
+    // Your code here
+  </script>
```

Comment:
Hi Andre, you JavaScript code is missing from this one.

Reply:
Hello Tim, I have updated my code!